src/tasks/daily.ts

```diff
@@ -365,6 +367,21 @@ export function configureSnojo(): void {
   }
 }
 
+function determineFreeEmbezzlerBanish(): void {
+  if (
+    gregLikeFightCount() < 120 &&
+    gregLikeFightCount() > 0 &&
+    have($item`mafia middle finger ring`) &&
+    !get("_mafiaMiddleFingerRingUsed") &&
+    have($skill`Snokebomb`) &&
+    get(`_snokebombUsed`) === 0
+  ) {
+    set("_garboUsingFreeEmbezzlerBanish", true);
+    return;
+  }
+  set("_garboUsingFreeEmbezzlerBanish", false);
```

Comment:
capture the value of the condition in the `if` in a variable, and `set` the pref to that variable, instead of using the early return

Reply:
120 is a magic number; either name it as a variable or comment as to how it came to be